CHANGELOG.md

```diff
@@ -14,6 +15,7 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 - Zero-crossing axis bounds (#1708)
 - Incorrect label placement on BarSeries with non-zero BaseValue (#1726)
+- Add checking distance for result between data points (#1736)
```

Comment:
If it is made optional, this will need to be an added feature rather than a fix.